apiserver/plane/bgtasks/export_task.py

```
@@ -10,13 +10,22 @@
 # Third party imports
 from celery import shared_task
 
+
 # Django imports
 from django.conf import settings
 from django.utils import timezone
 from openpyxl import Workbook
+from django.db.models import Value
```

Ruff (0.8.2)

`django.db.models.Value` imported but unused

Remove unused import: `django.db.models.Value`

(F401)